pkg/project/resource_project_membership_test.go

```diff
@@ -262,7 +263,7 @@ func TestAccProjectGroup(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "key", fmt.Sprintf("%s", params["project_key"])),
 					resource.TestCheckResourceAttr(resourceName, "display_name", name),
 					resource.TestCheckResourceAttr(resourceName, "description", "test description"),
-					resource.TestCheckNoResourceAttr(resourceName, "group"),
+					resource.TestCheckResourceAttr(resourceName, "group.#", "0"),
```

Comment:
The updated TF SDK doesn't like the old way any more. So have to change to this test.